src/tracers/blob_saver_tracer.rs

```diff
@@ -106,7 +110,23 @@ impl Tracer for BlobSaverTracer {
         let key = U256::from_big_endian(hash.as_bytes());
 
         self.blobs.insert(key, as_words);
+    }
+
+    fn after_execution(
+        &mut self,
+        _opcode: &Opcode,
+        _execution: &mut Execution,
+        _state: &mut VMState,
+    ) {
```

Comment:
nit: add `todo!()` inside of functions that should have logic inside.

Reply:
I've reimplemented the trait and now they have a default behavior of doing nothing, so we don't have to add them for every single tracer.